h/security/policy/_remote_user.py

```diff
@@ -16,7 +16,7 @@ def identity(self, request):
             return None
 
         user = request.find_service(name="user").fetch(user_id)
```

Comment:
`fetch_for_login` here if we move the check there?

Reply:
[Discussed on Slack](https://hypothes-is.slack.com/archives/C1MA4E9B9/p1716469004317419?thread_ts=1716394731.214939&cid=C1MA4E9B9): I'm gonna chicken-out from changing the security policies to call `fetch_for_login()` or changing the other `UserService.fetch*()` methods to filter out deleted users, because it's hard to reason through all the potentially far-reaching effects of these changes. Gonna stick with the more surgical change of just having the security policies read `user.deleted`